src/Language/PureScript/Graph.hs

```diff
@@ -0,0 +1,64 @@
+module Language.PureScript.Graph (graph) where
+
+import Prelude.Compat
```

Comment:
What does that mean? (I don't really have any strong opinions on this, I'm just wondering because it's not clear to me in the presence of the `qualified` keyword)

Reply:
I meant just sort them by alphabetical order and leave the unqualified ones separated with a space at the top while preferring explicit or qualified imports. It's the style most PureScript code uses and I'm convinced it's also a good idea in Haskell.

Not doing the explicit/qualified bit has also caused trouble with updating the Stackage resolver before and I'd prefer to avoid that in the future (because of sudden name clashes).

Reply:
@kritzcreek my import-style might not be clear, so to clarify what I'm doing:
- first I have an open `Prelude` import
- then `qualified` library imports
- then explicitly import names from libraries
- then `qualified` imports from the project
- then explicitly import names from the project

The point of this structure is that it makes clear separations between:
- imports from the project vs dependencies
- qualified vs not qualified imports

I can sort them alphabetically, but I'm still confused about why it's a problem to have some unqualified imports, as it's mostly just types and there's no open imports?

Reply:
Sorry, the explicit/qualified imports are just fine in here. I mixed talking about imports in general and the import structure in this file in my comment above.

I'll back off, don't worry about the imports.